src/common/browser.ts

```diff
@@ -0,0 +1,8 @@
+// Took this one-liner from: https://www.npmjs.com/package/browser-or-node
+export const isBrowser = typeof window !== 'undefined' && typeof window.document !== 'undefined'
+
+export const assertBrowser = (method: string): void => {
```

Comment:
I like this! It's nice to let people know why something isn't working as expected. But, maybe i missed it - are we not going to provide a node implementation as well?

Reply:
You didn't miss it. I just felt that was out of scope for this PR. While working on the filecoin stuff, the philosophy I've been taking with changes to `webnative` & the auth-lobby etc is: do the smallest change possible to allow what you need that isn't hacky and doesn't accrue tech debt.

So in otherwords, this is all I need for the `fil-cosigner` server, it allows developers to use `webnative` in Node (with some extra work) and Brian to right tests. Node defaults coming soon :grin: